docs/guide/usage.md

```diff
@@ -81,6 +81,42 @@ this path instead: import '~@kong/kongponents/dist/style.css' */
 </style>
 ```
 
+## SSR
+
+If you're using Kongponents in SSR mode, we advice [registering Kongponents as a Vue Plugin](#vue-plugin).
+
+Kongponents use external function to generate random strings that are mostly used in accessibility-related attributes. In server-side rendering, that will produce hydration mismatch because the function returns different values on server and client side. However, you can provide your own SSR-compatible function for generating random strings or ids through plugin options.
```

Comment:
```suggestion
Kongponents uses an external function to generate UUID strings that are mostly used in accessibility-related attributes. 

In server-side rendering, these unique ids will produce hydration mismatch errors because the function returns different values on the server and client. 

You can provide your own SSR-compatible function for generating random strings or ids through the Vue plugin options.
```